libs/components/lists/src/lib/modules/shared/filter-adapter/filter-adapter.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Observable } from 'rxjs';
+
+import { SkyFilterAdapterData } from './filter-adapter-data';
+
+/**
+ * Abstract service that provides an interface for components to integrate with a data manager.
+ * This allows various list and filter components to work with data managers without tight coupling.
+ * @internal
+ */
+export abstract class SkyFilterAdapterService {
```

Comment:
What's the reasoning for using "adapter" in the name, instead of going with the `SkyFilterStateService` like we discussed prior? I noticed that the JSDoc comments mention "data manager"; this service should be agnostic to whatever wants to use it, so I wouldn't say its an adapter for the data manager state, but its own thing: a service to manage filter state.

The filter bar implements the filter state service to alert any parent components of filter state changes.

The data manager directive provides the filter state service and creates the "adapter" or bridge between the data manager state service and the filter state service.